invenio_requests/records/mappings/v6/request_events/event-v1.0.0.json

```
@@ -0,0 +1,41 @@
+{
+  "mappings": {
+    "_doc": {
+      "dynamic": "strict",
+      "properties": {
+        "$schema": {
+          "type": "keyword"
+        },
+        "created": {
+          "type": "date"
+        },
+        "updated": {
+          "type": "date"
+        },
+        "uuid": {
+          "type": "keyword"
+        },
+        "version_id": {
+          "type": "keyword"
+        },
+
+        "id": {
+          "type": "keyword"
+        },
+        "type": {
+          "type": "keyword"
+        },
+        "request": {
+          "type": "keyword"
+        },
+        "request_id": {
+          "type": "keyword"
+        },
+        "extras": {
+          "type": "object",
+          "dynamic": true
```

I think we should skip indexing of the extras/payload: 
```suggestion
          "enabled": false
```

---

We won't be able to perform search on those which I think is fine for now.